theme/static/style/style.css

```diff
@@ -377,6 +483,21 @@ pre code {
 @media screen and (max-width: 991px){
     /* For tablets: */
 
+    /*navigation section*/
+    .topnav .col-4 img{
+        margin-top: 12.5px;
+        max-width: 200px;
```

Comment:
Replace the `px` units with `rem` or any other units that are commonly used in this stylesheet.